### PR TITLE
Assorted cleanups.

### PR DESCRIPTION
**Pull request by UncombedCoconut**

### CODE REVIEW

**Comment by bobbens** [comment]
Thanks!

File: src/faction.c [hunk 1]
```diff
@@ -452,22 +452,22 @@ void faction_addEnemy( int f, int o)
...
    if (!faction_isFaction(o)) { /* o is invalid */
-      WARN(_("addEnemy: %d is an invalid faction"), o);
+      WARN(_("Faction id '%d' is invalid."), o);
       return;
    }
 
    /* player cannot be made an enemy this way */
    if (f==FACTION_PLAYER) {
-      WARN(_("addEnemy: %d is the player faction"), f);
+      WARN(_("Faction id '%d' is invalid."), f);
```
Comment (onpon4) [inline src/faction.c:466]: This should still be warning that it's the player faction, shouldn't it?

🕵️

**Review by onpon4 — COMMENTED** [review]
Just to check, are those indications of what function the warnings are from redundant? (I think so but just checking.)

**Comment by UncombedCoconut** [comment]
> Just to check, are those indications of what function the warnings are from redundant? (I think so but just checking.)

Yes, ERR and WARN (only) include __func__, which expands to the function name. I'm guessing they weren't redundant when originally written, but they are now.

File: src/faction.c (hunk 1 above)
Comment (UncombedCoconut) [reply src/faction.c:466]: Oh no, good catch, thanks! Fixing now.

**Comment by bobbens** [comment]
Thanks!

**Pull request merged**